src/pages/graphql/announcing-our-new-graphql-bootcamp.md

```diff
@@ -9,7 +9,7 @@ order: 50
 
 At React GraphQL Academy, we’re keen to get more people into the world of the React ecosystem. Our expert developers have been coaching since the inception of the ecosystem, and our React and React Native training courses have gone down really well with developers and companies throughout the EU.
 
-But anyone who knows anything about React will tell you there’s so much more to sink your teeth into! That said, we’re thrilled to be bringing a brand new course to developers. Today, we’re announcing our new [GraphQL Bootcamp](https://reactgraphql.academy/graphql/training/bootcamp/?utm_medium=direct&utm_source=blog&utm_campaign=graphql_exp&utm_content=graphql_announcement).
+But anyone who knows anything about React will tell you there’s so much more to sink your teeth into! That said, we’re thrilled to be bringing a brand new course to developers. Today, we’re announcing our new [GraphQL Bootcamp](https://reactgraphql.academy/graphql/training/bootcamp/).
```

Comment:
@ric9176 internal links should not have the https and domain, otherwise, the Link component will treat them as a different site and won't do the HMTL5 history.pushState. Therefore a round trip to the server to load the new page-

Reply:
(/graphql/training/bootcamp/)  instead of (https://reactgraphql.academy/graphql/training/bootcamp/)

Reply:
makes sense, I didn't add these ;-) I'll sort this

Reply:
yes! I didn't add these links ;-) but I went through removing them too fast, I'll sort this!

Reply:
I already updated this and realized something when searching for others, it's true about the links for our site but when these are re-posted on Medium the links will break.